hips/draw/simple.py

```diff
@@ -45,20 +47,32 @@ class SimpleTilePainter:
     ----------
     geometry : `~hips.utils.WCSGeometry`
         An object of WCSGeometry
+    hips_survey : `~hips.HipsSurveyProperties`
+        HiPS survey properties
     tile : `HipsTile`
        An object of HipsTile
     """
 
-    def __init__(self, geometry: WCSGeometry, tile: HipsTile) -> None:
+    def __init__(self, geometry: WCSGeometry, hips_survey: HipsSurveyProperties, tile: HipsTile) -> None:
         self.geometry = geometry
+        self.hips_survey = hips_survey
         self.tile = tile
 
     @property
+    def dst(self) -> np.ndarray:
+        """Destination array for projective transform"""
+        return np.array(
+            [[self.hips_survey.tile_width - 1, 0],
```

Comment:
I'd suggest to create a local variable
```
width = self.hips_survey.tile_width
```
and then use it instead of having that lengthy expression 4 times.